content/blog/2023-10-05-october-Prism-launch/contents.lr

```diff
@@ -0,0 +1,22 @@
+title: Prism: The New Feinberg Repository for Global Dissemination of Research
+---
+pub_date: 2023-10-10
+---
+team: InvenioRDM
+---
+body:
+
+![](NEW_PICTURE_FILE_NEEDED.jpg)
```

Comment:
Maybe we could link to https://github.com/inveniosoftware/inveniosoftware.org/blob/master/assets/static/img/screenshots/nu_prism.png which is already in the repo (or copy in appropriate format to appropriate location)